cpp-ch/local-engine/local_engine_jni.cpp

```diff
@@ -535,10 +537,55 @@ JNIEXPORT void Java_org_apache_gluten_vectorized_CHStreamReader_nativeClose(JNIE
     LOCAL_ENGINE_JNI_METHOD_END(env, )
 }
 
+local_engine::SplitterHolder * buildAndExecuteShuffle(JNIEnv * env,
+        jobject iter,
+        const String & name,
+        const local_engine::SplitOptions& options,
+        jobject rss_pusher = nullptr
+        )
+{
+    auto * current_executor = local_engine::LocalExecutor::getCurrentExecutor();
+    chassert(current_executor);
+    local_engine::SplitterHolder * splitter = nullptr;
+    // handle fallback, whole stage fallback or partial fallback
```

Comment:
In which cases， current_executor will be nullptr or not? It's better to add a comment here

Reply:
done